mindsdb/integrations/handlers/portkey_handler/portkey_handler.py

```diff
@@ -0,0 +1,74 @@
+from typing import Dict, Optional
+
+import pandas as pd
+from portkey_ai import Portkey
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.utilities import log
+
+from mindsdb.integrations.utilities.handler_utils import get_api_key
+
+logger = log.getLogger(__name__)
+
+
+class PortkeyHandler(BaseMLEngine):
```

Comment:
It would be great if we can add a `create_engine()` method to determine the validity of the API key. Otherwise, the `CREATE ENGINE` command will succeed and the validity of the key will only be evaluated when running `CREATE MODEL`.

Reply:
The key cannot be validated until the model is created, there is no default key stored in portkey since it's not a traditional provider, it routes to different providers based on user configuration